compiler/modules/front.syntax/token.bal

```diff
@@ -236,12 +236,14 @@ class Tokenizer {
         }
     }
 
-    function peek() returns VariableTokenCode|FixedToken? {
+    // Peeks the next token. When skipQualIdent is passed, considers qualified identifier as a single token,
+    // ie if current token is the first identifier of a qualified identifier, returns the token after the second identifier.
+    function peek(boolean skipQualIdent) returns VariableTokenCode|FixedToken? {
```

Comment:
I think this should default to `false`, since that is just a plain `peek` (and then use this default).